org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/labeling/PureXbaseDescriptionLabelProvider.java

```diff
@@ -8,8 +8,8 @@
 /**
  * Provides labels for IEObjectDescriptions and IResourceDescriptions.
  * 
- * See https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#label-provider
+ * https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#label-provider
```

Comment:
Why remove "See" here? Maybe even use `@see`?

Reply:
this is what the generator "generates"

Reply:
this is what the generator generates